zipline-security/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+import com.vanniktech.maven.publish.JavadocJar
+import com.vanniktech.maven.publish.KotlinMultiplatform
+import com.vanniktech.maven.publish.MavenPublishBaseExtension
+import org.jetbrains.kotlin.gradle.plugin.PLUGIN_CLASSPATH_CONFIGURATION_NAME
+
+plugins {
+  kotlin("multiplatform")
+  kotlin("plugin.serialization")
+  id("org.jetbrains.dokka")
+  id("com.vanniktech.maven.publish.base")
+  id("binary-compatibility-validator")
+}
+
+kotlin {
+  jvm()
+
+  js {
+    nodejs()
+  }
+
+  linuxX64()
+  macosX64()
+  macosArm64()
+  iosArm64()
+  iosX64()
+  iosSimulatorArm64()
+  tvosArm64()
+  tvosSimulatorArm64()
+  tvosX64()
+
+  applyDefaultHierarchyTemplate()
+
+  sourceSets {
+    val commonMain by getting {
+      dependencies {
+        api(project(":zipline"))
+      }
+    }
+
+    val hostMain by creating {
+      dependsOn(commonMain)
+    }
+
+    val nativeMain by getting {
+      dependsOn(hostMain)
+    }
+
+    val jvmMain by getting {
+      dependsOn(hostMain)
+    }
+  }
+
+  // TODO(jessewilson): move this to a common build module, shared with zipline-testing.
```

Comment:
@swankjesse fyi that we left this as a follow up

Reply:
👍🏻